composer.json

```diff
@@ -23,12 +23,13 @@
         "php-http/httplug": "^1.0",
         "php-http/message-factory": "^1.0",
         "php-http/discovery": "^1.0",
-        "symfony/options-resolver": "^3.0 || ^4.0"
+        "symfony/options-resolver": "^3.0 || ^4.0",
+        "php-http/curl-client": "^1.7",
```

Comment:
Aanpassingen terugdraaien.
De dependencies stonden goed in composer.json. De implementatie wordt in het project gekozen